uqcsbot/text.py

```diff
@@ -273,6 +286,44 @@ async def zalgo_command(self, interaction: discord.Interaction, text: str):
 
         await interaction.response.send_message(self.zalgo_common(text))
 
+    def rot_13_cipher(self, text: str) -> str:
+        result = ""
+        for c in text:
+            if "a" <= c <= "m" or "A" <= c <= "M":
+                result += chr(ord(c) + 13)
+            elif "n" <= c <= "z" or "N" <= c <= "Z":
+                result += chr(ord(c) - 13)
+            else:
+                result += c
+        return result
+
+    @app_commands.command(name="rot_13")
+    @app_commands.describe(text="Input text")
+    @yelling_exemptor()
+    async def rot_13_command(self, interaction: discord.Interaction, text: str):
+        """
+        Encodes the given text with the cunning ROT13 Cipher
+        """
+        await interaction.response.send_message(self.rot_13_cipher(text))
+
+    ##    async def rot_13_context(
```

Comment:
I think deciding between these two options really comes down to which gets used more often. As we don't have much use of ROT13 already, I can't really tell. I would think that having it ephemeral would be preferable, as people already have the command if they want everyone  to see it.

Reply:
The non-ephemeral version was written before I found out about the limit of five context commands.